Library/Homebrew/test/.rubocop.yml

```diff
@@ -5,3 +5,8 @@ inherit_from:
 RSpec:
   Include:
     - ./*
+
+RSpec/MultipleDescribes:
```

Comment:
Is this to separate e.g. `"Homebrew.install_args"` and `"brew install", :integration_test`? I wonder if we could still combine them but have a different `context` (for the `integration_test`) and a base-level `describe`.  `brew install has parseable arguments` or similar would work for me for a description.

Reply:
Let me take another look. I think the `it_behaves_like "parseable arguments"` can be modified if needed to match a new structure. That may have been where the issue was when i tried before.

Reply:
Separately, what exactly does adding `:integration_test` do? I know they're slower, but is that because it's actually running the brew commands using the `brew` method or just because of the integration test marking?

Would it be bad to have the integration test metadata applied to the `it_behaves_like "parseable arguments"` as well, even though those tests aren't integration tests?

Reply:
How about something like this:

```ruby
describe "brew --cache" do
  it_behaves_like "parseable arguments"

  context "when running", :integration_test do
    it "prints all cache files for a given Formula" do
      expect(true).to eq false # let's not commit this, though 😅
    end
  end
end
```

And then, the failed output looks like:

```
rspec ./test/cmd/--cache_spec.rb:10 # brew --cache when running prints all cache files for a given Formula
rspec ./test/cmd/--cache_spec.rb:7 # brew --cache behaves like parseable arguments can parse arguments
```

Not a huge fan of `behaves like parseable arguments can parse arguments` but that's present already so it's not worse.

Reply:
> How about something like this:

Looks great!



> Not a huge fan of `behaves like parseable arguments can parse arguments` but that's present already so it's not worse.

Yeh. Lovely to fix but not a blocker by any means.



> Separately, what exactly does adding `:integration_test` do? I know they're slower, but is that because it's actually running the brew commands using the `brew` method or just because of the integration test marking?

Essentially it spins up a totally new `brew` Ruby process so it has about a ~1.5-2s overhead for every test 😭

Reply:
> Yeh. Lovely to fix but not a blocker by any means.

Plus, if this test fails, we've got much bigger problems so I'm not super worried about the messaging 😅

> Essentially it spins up a totally new `brew` Ruby process so it has about a ~1.5-2s overhead for every test 😭

Yeah, I figured something like that. I'll manage. I think it should be able to be applied to a specific example rather than a whole example group, so that's probably what I'll do. If not, I'll just nest them within another `describe` or `context` block.